web_interface/blueprints/api_v3.py

```
@@ -6425,9 +6441,12 @@ def list_plugin_assets():
 def get_logs():
     """Get system logs from journalctl"""
     try:
+        if not _SUDO or not _JOURNALCTL:
+            return jsonify({'status': 'error', 'message': 'sudo or journalctl not found on this system'}), 503
         # Get recent logs from journalctl
         result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)